pixart.js

```diff
@@ -1 +1,37 @@
+// commit 3 function
+function clickSquare(){
+      $(this).css("background", "green");
+    }
 
+function chooseMyC(){
+  var matchC=$('#color-field').val();
+  $(this).css("background", matchC);
+}
+//commit 1
+function updateC(evt)
+{
+ evt.preventDefault();
+ var matchC=$('#color-field').val();
+  $('div.brush').css("background", matchC);
+}
+
+
+  var buttonNode=$('#set-color');
+  buttonNode.click( updateC);
+
+//commit 2, commit 6
+for(var i=0; i<8000; i++){
+  $('body').append('<div class="square"> </div>');
+}
+
+// commit 3
+var squareNode=$('div.square');
+squareNode.click(clickSquare);
```

Comment:
We want to draw with mouseOver rather than having to click each pixel. How can we do that?